pom.xml

```diff
@@ -150,6 +150,9 @@
   <properties>
     <scm.repository>talend/component-runtime</scm.repository>
 
+    <maven.compiler.source>17</maven.compiler.source>
```

Comment:
When do we plan to use this new component-runtime in Studio?
We can't use it before R2025-02, we should support Java 8 and Java11 before R2025-02

Reply:
Yeah, aware on the delays. We may postpone to next rel.